tecan_od_analyzer/tecan_od_analyzer.py

```diff
@@ -475,7 +478,8 @@ def linear(x, a, b):
             df_gr_comp["Correlation"]
         df_gr_comp_out = df_gr_comp_out.append(df_gr_comp)
 
-    plt.savefig("lm_volume_loss.png", dpi=250)
+    # plt.savefig("lm_volume_loss.png", dpi=250)
```

Comment:
I would recommend allowing the user to choose whether they want .png or .svg output by introducing an additional option in the command line arguments.